python/python-quiz.md

```diff
@@ -1396,3 +1396,39 @@ num_list.remove(2)
 - [x] list(reversed(range(1,10)))
 - [ ] list(range(10,1,-1))
 - [ ] list(reversed(range(1,11)))
+
+### Q1. What is the output of this code? (NumPy has been imported as np.)
```

Comment:
```suggestion
### Q104. What is the output of this code? (NumPy has been imported as np.)
```

Reply:
FYI the numbers already should be different, please update them after you solve merge conflicts, thanks